holopy/core/tests/test_io.py

```diff
@@ -33,12 +33,17 @@
 import holopy as hp
 from holopy.core import load, save, load_image, save_image
 from holopy.core.errors import NoMetadata
+from holopy.core.io import load_average, get_example_data_path
 from holopy.core.process import normalize
 from holopy.core.metadata import get_spacing, copy_metadata
 from holopy.core.holopy_object import Serializable
 from holopy.core.tests.common import (
     assert_obj_close, assert_read_matches_write, get_example_data)
 
+import memory_profiler
```

Comment:
Tests are currently failing since travis doesn't have this installed. Add it to `.travis.yml`.

Reply:
Hmm... I disagree. I don't think we need ```memtest``` as a dependency for 1 unit test. I would rather do one of the following:
1.  Skip the unittest if memtest isn't installed.
2.  Remove the unit test which checks memory.

Reply:
I wasn't suggesting we make it a dependency, just that we get travis to install it before running tests. I agree removing the test would be ok, since it's not testing functionality in the usual way.

Reply:
This is fixed by 02a7155